core/src/types.ts

```diff
@@ -88,6 +89,11 @@ export declare interface RegularCalleeConfig {
     route: string[];
 }
 
+export declare interface UniswapV3AutoRouterCalleeConfig {
+    callee: RegularCalleeConfig['callee'];
```

Comment:
Since you're extending `RegularCalleeConfig['callee']` it's not only `UniswapV3`
```suggestion
export declare interface AutoRouterCalleeConfig {
    callee: RegularCalleeConfig['callee'];
```